### PR TITLE
attempting to style the checkbox

### PR DESCRIPTION
**Pull request by beccaklam**

### CODE REVIEW

File: less/pages/safety/safety.less [hunk 1]
```diff
@@ -105,6 +105,26 @@
...
       height: 1rem;
     }
+    input[type=checkbox] {
+      display: none;
+    }
+    label:before {
+      content: "";
+      display: inline-block;
+      width: 24px;
+      height: 24px;
```
Comment (ScottDowne) [inline less/pages/safety/safety.less]: Do we want to do height/width in rems?

File: less/pages/safety/safety.less [hunk 2]
```diff
@@ -105,6 +105,26 @@
...
+      position: relative;
+      background-color: #B94FBB;
+      border: 1px solid white;
+      margin-right: 7px;
```
Comment (ScottDowne) [inline less/pages/safety/safety.less]: Could also do these in rems.

File: src/components/safety/signup-form.js [hunk 3]
```diff
@@ -125,12 +125,12 @@ var SignupForm = React.createClass({
...
         <ErrorMessage>{this.state.emailInputError}</ErrorMessage>
-        <label>
-          <input className="privacy-checkbox" type="checkbox" onChange={this.onPrivacyChange} value={this.state.privacyCheckboxChecked}/>
-          <span>
-            I'm ok with mozilla handling my info as explained in this <a href="">privacy notice</a>
-          </span>
-        </label>
+        <div>
+          <input className="privacy-checkbox" type="checkbox" id="c1" onChange={this.onPrivacyChange} value={this.state.privacyCheckboxChecked}/>
+          <label htmlFor="c1">
```
Comment (ScottDowne) [inline src/components/safety/signup-form.js]: What does `c1` stand for? Or can we call it something a bit more descriptive?

Comment (beccaklam) [inline src/components/safety/signup-form.js]: Oh it just means checkbox 1, but we can call it checkbox.

File: less/pages/safety/safety.less (hunk 1 above)
Comment (beccaklam) [inline less/pages/safety/safety.less]: Sure what is your base rem? As in what should I divide the 24 by?

File: src/components/safety/signup-form.js (hunk 3 above)
Comment (ScottDowne) [inline src/components/safety/signup-form.js]: Let's call it privacy-checkbox

File: less/pages/safety/safety.less (hunk 1 above)
Comment (ScottDowne) [inline less/pages/safety/safety.less]: I dunno, I set this to 1.1rem locally, and seemed to work and is a pretty nice simple rem.

File: less/pages/safety/safety.less (hunk 2 above)
Comment (ScottDowne) [inline less/pages/safety/safety.less]: Could make this `margin-right: 0.3rem;` or `margin-right: 0.32rem;` if you want to get precise.

**Comment by ScottDowne** [comment]
Merged in https://github.com/mozilla/advocacy.mozilla.org/pull/333

**Pull request closed without merging**